en/peripherals/gripper.md

```diff
@@ -0,0 +1,107 @@
+
+Grippers are mechanical devices that can be integrated with an unmanned vehicle to hold (grip) and release payloads.
+
+PX4 allows grippers to triggered automatically in [Payload Delivery Missions](../flying/package_delivery_mission.md) or manually using a Joystick.
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+:::note
+A gripper can also be configured as a [generic RC or MAVLink actuator](../payloads/README.md#actuator-control-with-rc).
+This allows it to be used manually via an RC Controller, but not in missions or on a Joystick.
+:::
+
+## Supported Grippers
+
+There are many different gripper mechanisms ("jaws", "fingers", "electromagnets") and interfaces (PWM, CAN, MAVLink, and so on).
+
+PX4 supports grippers that have simple triggers to hold and release, and that use the following interfaces (see linked documents for details):
+
+- [PWM Servo Gripper](gripper_servo.md) - Grippers connected to autopilot PWM outputs
+- **MAVLink Gripper** (Untested) - Grippers that support the [MAV_CMD_DO_GRIPPER](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_GRIPPER) MAVLink command.
+
+:::note
+The [Roboclaw driver](../modules/modules_driver.md#roboclaw) isn't supported as a Gripper yet.
+However in the future it may get supported as a Gripper.
+:::
+
+## Using a Gripper
+
+For information on using a gripper in missions see [Payload Delivery Missions](../flying/package_delivery_mission.md).
+
+You can trigger the gripper manually if you've mapped `gripper open` and `gripper close` buttons in the [QGC Joystick Configuration](#qgc-joystick-configuration).
+Note that if you press **Grab** button while the gripper is opening, it will automatically abort releasing behavior and go to the Close position, effectively cancelling the release command.
+If you do this in a mission while the release is actually happening then the [delivery will be cancelled](../flying/package_delivery_mission.md#manual-control-of-gripper-in-missions).
+
+MAVLink applications, such as ground stations, can also control the gripper using the [MAV_CMD_DO_GRIPPER](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_GRIPPER) MAVLink command.
+
+
+## PX4 Configuration
+
+### Package Delivery Configuration
+
+PX4 Gripper support is tied to the package delivery feature, which must be enabled and configured in order to be able to use a gripper.
+
+1. Set [`PD_GRIPPER_EN`](../advanced_config/parameter_reference.md#PD_GRIPPER_EN) parameter to 1 (reboot required after change).
+1. Set [`PD_GRIPPER_TYPE`](../advanced_config/parameter_reference.md#PD_GRIPPER_TYPE) to match your Gripper.
+   For example, set to `Servo` for a [Servo Gripper](gripper_servo.md).
+
+### Gripper Actuator Mapping
+
+To enable the output of the Gripper, set the Function of the output port where the gripper is connected to as `Gripper` in the [Actuators](../config/actuators.md#actuator-outputs) tab.
+
+<!-- Replace this with actuator diagram -->
```

Comment:
@junwoo091400 Have we got a screenshot of a gripper setup in control allocation? Would be good here.

Reply:
Added!